src/tools/create-file-stasher.js

```diff
@@ -75,6 +75,11 @@ const createFileStasher = (input) => {
       return ZapierPromise.reject(new Error('rpc is not available'));
     }
 
+    const isRunningOnHydrator = _.get(input, '_zapier.event.method', '').indexOf('hydrators.') === 0;
+    if (!isRunningOnHydrator) {
+      return ZapierPromise.reject(new Error('Cannot stash files outside an hydration function/method.'));
```

Comment:
Is there anything we need to do so that tests trying to check a hydrate method don't break? My read of this is that tests that call a hydration method directly would need to define the `_zapier.event.method`? Is that right, or will that get auto generated during testing? If not, wonder if we could make the test harness smart enough to add that in.